src/style.css

```diff
@@ -57,17 +56,18 @@
   outline: none;
 }
 
-.form[data-qa="small"] > .form__input {
+.form--small .form__input {
   height: 42px;
   font-size: 14px;
   padding-left: 33px;
 }
 
-.form[data-qa="small"] .form__btn {
-  width: 11px;
-  height: 11px;
-  background-size: 11px 11px;
-  background-repeat: no-repeat;
+.form--small .form__btn {
```

Comment:
Sorry I have missed that. According to checklist you should try your best not to increase css selector specificity,
So instead of **.form--small .form__btn** you can add new class to style thise elements :)